smart-table-module/js/Table.js

```diff
@@ -60,7 +68,8 @@
 
                 var dataRow, oldValue;
 
-                if ((!angular.isArray(array)) || (selectionMode !== 'multiple' && selectionMode !== 'single')) {
+                if ((!angular.isArray(array)) || (selectionMode !== 'multiple' 
+                    && selectionMode !== 'single' && selectionMode !== 'customMultiple')) {
```

Comment:
I did not use isModeMultiple() because this function takes a parameter "selectionMode" instead of using scope.selectionMode. I'm not sure exactly why they chose to implement it this way, but there may be a reason, so I did not want to change it.